nohwookjin/src/components/Main.tsx

```diff
@@ -0,0 +1,82 @@
+import React, { useState } from "react";
+import styled from "styled-components";
+
+import Jap from "../components/Jap";
+import Korean from "../components/Korean";
+import Cafe from "../components/Cafe";
+
+const Main = () => {
+  const [cov, setCov] = useState<boolean>(false);
+  const [cov2, setCov2] = useState<boolean>(false);
+  const [cov3, setCov3] = useState<boolean>(false);
+
+  const onClickHandler = () => {
+    if (cov2 === true || cov3 === true) {
+      setCov2(false);
+      setCov3(false);
+    }
+    setCov(!cov);
+  };
+
+  const onClickHandler2 = () => {
+    if (cov === true || cov3 === true) {
+      setCov(false);
+      setCov3(false);
+    }
+    setCov2(!cov2);
+  };
+
+  const onClickHandler3 = () => {
+    if (cov === true || cov2 === true) {
+      setCov(false);
+      setCov2(false);
+    }
+    setCov3(!cov3);
+  };
+
+  return (
+    <StyledSection>
+      <StyledSideBar>
+        <StyledBtn onClick={onClickHandler}>•일식</StyledBtn>
+        <StyledBtn onClick={onClickHandler2}>•한식</StyledBtn>
+        <StyledBtn onClick={onClickHandler3}>•카페</StyledBtn>
+      </StyledSideBar>
+      <article>
+        {cov === true ? <Jap /> : null}
```

Comment:
```suggestion
        {cov === true && <Jap />}
```

이런 경우에는 `&&`  연산자를 이용하는 것이 권장돼요!

Reply:
아무 생각없이 삼항연산자 쓰는게 습관이 되버렸네용.. 고치겠습니다!